packages/web3-providers-ws/package.json

```diff
@@ -8,6 +8,6 @@
     "dependencies": {
         "underscore": "1.9.1",
         "web3-core-helpers": "1.0.0-beta.37",
-        "websocket": "1.0.28"
+        "websocket": "github:frozeman/WebSocket-Node#browserifyCompatible"
```

Comment:
It looks like you've switched this back to the version that was incorrectly packed. That version includes a `.git` directory in the published module, which results in npm errors in certain cases (e.g. when running `npm install` a second time).

A few people in this PR already expressed relief at this error going away; it was the part I was most looking forward to about this release personally. Could you elaborate on why this update was reverted? I'd be happy to help out if you ran into an issue you don't have time to investigate.